src/Experiment/tests/BlankAssemblyTest.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Emit.Experimental;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace Experiment.Tests
+{
+    public class BlankAssemblyTest
+    {
+        private readonly AssemblyName _name = new AssemblyName("MyBlankAssembly");
+        [Fact]
+        public void GenerateBlankAssemblyReadToDisk()
+        {
+            const String _fileName = "BlankAssembly.dll";
+            AssemblyBuilder assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly(_name, System.Reflection.Emit.AssemblyBuilderAccess.Run);
+            assemblyBuilder.Save(_fileName);
+            Reader(_fileName);//It seems that sometimes ILSpy will point out a DLL is malformed but MetadataReader doesn't throw an error (MetadataLoadContext rather?) .
+        }
+       
+
+        private void Reader(String fileName)
```

Comment:
Should this be something like:
`private static bool TryLoadAssembly(string filePath)`